terraform/modules/security/main.tf

```diff
@@ -0,0 +1,48 @@
+terraform {
+  required_providers {
+    aws = {
+      source                = "hashicorp/aws"
+      version               = "~> 4.0"
+      configuration_aliases = [aws.us-east-1]
+    }
+  }
+}
+
+resource "aws_securityhub_finding_aggregator" "securityhub" {
+  depends_on   = [module.securityhub_local]
+  linking_mode = "ALL_REGIONS"
+}
+
+module "securityhub_local" {
```

Comment:
"Local" here is "eu-west-2" right?

Reply:
With how the module works, its whatever region you set